src/server/FileParser/Parsers/AureliaHtmlParser.ts

```diff
@@ -87,15 +87,22 @@ export class AureliaHtmlParser {
     return bindings;
   }
 
-  private getStringInterpolationBindings(fileContent) {
+
+  private getStringInterpolationBindings(fileContent: string, document: TagDefinition[]) {
+    const tags = document.filter(tag => {
+      const isDontParseTags = (tag.name === 'template') || (tag.name === 'require');
+      const isStartTag = !isDontParseTags && tag.startTag;
+      return (!isDontParseTags && tag.startTag) // omit closing tags
+    });
+
     let bindings = [];
     const aureliaParser = new Parser();
     const interpolationRegex = /\$\{(.*)\}/g;
     var match;
     while (match = interpolationRegex.exec(fileContent)) {
       bindings.push({
         value: match[0],
-        bindingData: aureliaParser.parse(match[1])
+        // bindingData: aureliaParser.parse(match[1]) // some files got parsing errors

```

Comment:
This line causes some issues in files with nested string interpolations 
(compare #101)

So for the sake of 'goto' feature, I commented that out.
Could uncomment and have few files not work, but I'd rather have goto in 99% of my files